packages/ui/src/components/primitives/Contributors.tsx

```diff
@@ -0,0 +1,79 @@
+import Image from 'next/image';
+import { ArrowTopRightOnSquareIcon } from '@heroicons/react/24/solid';
+import { FollowCursor } from './Follow';
+import { GitHubIcon } from './base';
+
+export interface ContributorData {
+	name: string;
+	description: string;
+	avatar: string;
+	socials: Record<string, string>;
+}
+
+const linkMap: { [key: string]: string | React.FC<{ className?: string }> } = {
```

Comment:
why not use a Map here? like the actual JS class

Reply:
Done :+1: